openedx/features/edly/tests/test_models.py

```diff
@@ -3,8 +3,14 @@
 """
 from django.test import TestCase
 
-from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
-from openedx.features.edly.tests.factories import EdlyOrganizationFactory, EdlySubOrganizationFactory, SiteFactory
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization, EdlyUserProfile
+from openedx.features.edly.tests.factories import (
+    EdlyOrganizationFactory,
+    EdlySubOrganizationFactory,
+    EdlyUserProfileFactory,
+    SiteFactory,
+    EdlyUserFactory
```

Comment:
fix imports order, also add comma , at the end even for last element so that we can reduce git diff to one line in case of addition of new import

Reply:
Done